src/Path/Include.hs

```diff
@@ -833,22 +846,15 @@ normalizeWindowsSeps :: FilePath -> FilePath
 normalizeWindowsSeps path = normLeadingSeps ++ normalizeAllSeps rest
   where (leadingSeps, rest) = span FilePath.isPathSeparator path
         normLeadingSeps = replicate (min 2 (length leadingSeps)) FilePath.pathSeparator
-
--- | Normalizes seps only at the beginning of a path.
-normalizeLeadingSeps :: FilePath -> FilePath
-normalizeLeadingSeps path = normLeadingSep ++ rest
-  where (leadingSeps, rest) = span FilePath.isPathSeparator path
-        normLeadingSep = replicate (min 1 (length leadingSeps)) FilePath.pathSeparator
-
--- | Normalizes seps only at the end of a path.
-normalizeTrailingSeps :: FilePath -> FilePath
-normalizeTrailingSeps = reverse . normalizeLeadingSeps . reverse
+#endif
 
 -- | Applies platform-specific sep normalization following @FilePath.normalise@.
 normalizeFilePath :: FilePath -> FilePath
-normalizeFilePath
-  | IS_WINDOWS = normalizeWindowsSeps . FilePath.normalise
-  | otherwise  = normalizeLeadingSeps . FilePath.normalise
+#if IS_WINDOWS
```

Comment:
Same question here: Why?
If you're going to do this, it makes sense to put the type signature inside the if statement so that it's easier for formatters to figure out.

Reply:
I changed that in the recent commits.